### ISSUE TITLE
Differentiate installation method

### ISSUE THREAD

**Issue by raffaeleflorio** [raffaeleflorio/qubes-url-redirector#38]
Differentiate installation method for AppVM and TemplateVM.

**Comment by coyotebush** [comment]
I presume this refers to installing the components (extension and native application) in system-level vs user-level directories. Currently they're split between those locations.

Relevant documentation:
https://extensionworkshop.com/documentation/publish/distribute-sideloading/#linux
https://developer.chrome.com/docs/extensions/how-to/distribute/install-extensions#preference-linux
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_manifests#manifest_location
https://developer.chrome.com/docs/extensions/develop/concepts/native-messaging#native-messaging-host-location

User-level installation for Chrome/ium looks like the hardest combination: there's no documented user-level Linux location for the preferences file, and the CRX file is an additional component to consider.

**Comment by coyotebush** [comment]
A fourth component I missed is the `qvm-open-in-vm-we.py` script. Since the native messaging manifest needs to contain an absolute path to that script, I think the native messaging manifest needs to have different contents depending on installation location.

I see you had been working on this, for instance in https://github.com/raffaeleflorio/qubes-url-redirector/commit/e57aa893e731b2a661b8f7a001e23b3dad180e14.

In summary:
* TemplateVM is easy: #48
* Firefox in AppVM seems possible but slightly involved
* Chrome/ium in AppVM may require users to do something like manually install the extension, symlink the extensions directory, or launch the browser with special flags
